src/main/java/seedu/duke/storage/StorageTasks.java

```diff
@@ -12,6 +12,10 @@
 
 //@author swatim
 public class StorageTasks {
+    //author ngnigel99 - create in folder
+    public static final String folderName = "tasksdata/";
+    public static final String fileName   = "scheduleTasks.txt";
+    public static final String filePath = folderName + fileName;
```

Comment:
@swatimahadevan I added these variables instead of using a constant, so your data would be organised into `tasksdata/` folder. feel free to change the name

Reply:
#41